StyleSheets/StyleSheets/SearchBarStyleSheet.swift

```diff
@@ -0,0 +1,66 @@
+import UIKit
+
+open class SearchBarStyleSheet: ViewStyleSheet<UISearchBar> {
+    public var height: CGFloat
+    public var showsCancelButton: Bool
+    public var searchTextPositionAdjustment: UIOffset
+    public var keyboardType: UIKeyboardType
+    public var returnKeyType: UIReturnKeyType
+    public var enablesReturnKeyAutomatically: Bool
+
+    public init(
+        backgroundColor: UIColor = .white,
+        cornerRadius: CGFloat = 10,
+        height: CGFloat = 36,
+        showsCancelButton: Bool = false,
+        searchTextPositionAdjustment: UIOffset = UIOffset(horizontal: 8, vertical: 0),
+        keyboardType: UIKeyboardType = .default,
+        returnKeyType: UIReturnKeyType = .search,
+        enablesReturnKeyAutomatically: Bool = true
+        ) {
+        self.height = height
+        self.showsCancelButton = showsCancelButton
+        self.searchTextPositionAdjustment = searchTextPositionAdjustment
+        self.keyboardType = keyboardType
+        self.returnKeyType = returnKeyType
+        self.enablesReturnKeyAutomatically = enablesReturnKeyAutomatically
+        super.init(backgroundColor: backgroundColor, cornerRadius: cornerRadius)
+    }
+
+    open override func apply(to element: UISearchBar) {
+        super.apply(to: element)
+        element.setTextInputBackgroundColor(color: backgroundColor ?? .clear,
+                                            height: height,
+                                            cornerRadius: cornerRadius)
+        element.showsCancelButton = showsCancelButton
+        element.searchTextPositionAdjustment = searchTextPositionAdjustment
+        element.keyboardType = keyboardType
+        element.returnKeyType = returnKeyType
+        element.enablesReturnKeyAutomatically = enablesReturnKeyAutomatically
+        element.barTintColor = tintColor
+        // remove system background
+        element.backgroundColor = .white
+        element.backgroundImage = UIImage()
```

Comment:
This should be a property of the stylesheet IMO as some may want to customise it as well

Reply:
I'm not sure, this is needed just to make these weird lines go away. Here how it looks without it.
<img width="378" alt="screen shot 2018-11-08 at 15 38 14" src="https://user-images.githubusercontent.com/1488293/48209143-5497ae80-e36c-11e8-83ad-49915a609ecc.png">

The background colour can be configured though so I'll adjust style to configure it separately for text field.

Reply:
ok